### PR TITLE
Release spark-hofs v0.5.0

### PR DESCRIPTION
**Pull request by yruslan**

### CODE REVIEW

**Comment by yruslan** [comment]
I see the release plugin removes license from `version.sbt`. Oh well, I don't think it is needed there 😄

File: README.md [hunk 1]
```diff
@@ -5,36 +5,50 @@ only via textual representation of Spark SQL.
...
-|       0.2.0        |     2.11      |     2.4.1       |
-|       0.3.x        |     2.11      |     2.4.2       |
-|       0.4.x        |  2.11, 2.12   |     2.4.3+      |
+| spark-hofs version | Scala version | Spark version |
+|:------------------:|:-------------:|:-------------:|
+|       0.1.0        |     2.11      |     2.4.0     |
+|       0.2.0        |     2.11      |     2.4.1     |
+|       0.3.x        |     2.11      |     2.4.2     |
+|       0.4.x        |  2.11, 2.12   |     2.4.x     |
+|       0.5.x        |  2.12, 2.13   |     3.x+      |
```
Comment (Zejnilovic) [inline README.md]: missing scala version 2.11 on 0.5?

Comment (yruslan) [reply README.md]: Yes, good spot!

Comment (yruslan) [reply README.md]: Actually, found another issue in that table. Fixed it.

**Review by Zejnilovic — APPROVED** [review]

**Pull request merged**